docs/modules/localstack.md

```diff
@@ -1,6 +1,6 @@
 # Localstack Module
 
-Testcontainers module for the Atlassian's [LocalStack](https://github.com/localstack/localstack), 'a fully functional local AWS cloud stack'.
+Testcontainers module for [LocalStack](http://localstack.cloud/), 'a fully functional local AWS cloud stack', to develop and test your cloud and serverless apps offline.
```

Comment:
```suggestion
Testcontainers module for [LocalStack](http://localstack.cloud/), 'a fully functional local AWS cloud stack', to develop and test your cloud and serverless apps without actually using the cloud.
```

Maybe rephrase like this? Since e.g. the container downloads certificates on startup (something I learned through the recent incident with the expired certificate), so you still need to be online.